src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPage.tsx

```diff
@@ -18,6 +18,7 @@ interface Props {
   newDesign?: boolean; // TODO EES-3517 flag
   paging?: Paging | null; // TODO EES-3517 won't be optional or null
   publications?: PublicationListSummary[]; // TODO EES-3517 won't be optional
+  search?: string;
   sortBy?: PublicationSortOption;
```

Comment:
We've talked about adding a 'relevance' sort option which becomes the default when using a search term.

I don't think it makes sense to order by any of the other sort options (title or release published date) when using search, or at least you'd expect results to be sorted by relevance by default.

The back-end API has support for this and also 'relevance' is the default option used by the API if a search term is present but no sort is specified.

See the description of https://github.com/dfe-analytical-services/explore-education-statistics/pull/3635

Reply:
Completely forgot about that, sorry! I'll have a chat with Marv about it.

Reply:
Chatted to Marv - I'm going to change it to add the relevance option and select it by default when you have a search term.